server/issue.go

```diff
@@ -540,6 +540,91 @@ func (p *Plugin) getIssueAsSlackAttachment(ji Instance, jiraUser JIRAUser, issue
 	return parseIssue(issue), nil
 }
 
+func (p *Plugin) assignJiraIssue(mmUserId, issueKey, assignee string) (string, error) {
+	ji, err := p.currentInstanceStore.LoadCurrentJIRAInstance()
+	if err != nil {
+		return "", err
+	}
+
+	jiraUser, err := ji.GetPlugin().userStore.LoadJIRAUser(ji, mmUserId)
+	if err != nil {
+		return "", err
+	}
+
+	jiraClient, err := ji.GetJIRAClient(jiraUser)
+	if err != nil {
+		return "", err
+	}
+
+	// required minimum of three letters in assignee value
+	minChars := 3
+	if len(assignee) < minChars {
+		errorMsg := fmt.Sprintf("`%s` contains less than %v characters.", assignee, minChars)
+		return errorMsg, nil
+	}
+
+	// check for valid issue key
+	_, _, err = jiraClient.Issue.Get(issueKey, nil)
```

Comment:
Can this check be skipped? I'd imagine the subsequent request(s) would fail on a non-existing ticket anyway...

Reply:
@levb, this does get checked later it the code if a non-existent issue key is provided.  Unfortunately, the message returned by `jiraClient.Issue.UpdateAssignee()` is generic.  In the image below, you can see the two cases.  The first is where I handle the err and the second is the return value from Jira. 

![image](https://user-images.githubusercontent.com/7575921/60838761-17045300-a191-11e9-9e1c-dfe33754c87e.png)